### DISCUSSION TITLE
What is the cmake option used of the official pre-built binaries?

### DISCUSSION THREAD

**Discussion by lingchengfenzi** [ClickHouse/ClickHouse#29986] [Q&A] [answered]
I want use another compiler to compile clickhouse. I hope use the options which the official compile options to compile it. Who can tell me that? Thanks.

**Comment by alexey-milovidov** [comment]
We have public build logs. Go to the commit, click on the "Build Check" and you will find the link to build logs.

**Reply by lingchengfenzi** [reply]
Thanks for your answer. But I failed to find the "Build Check". Can you give a sample of the build logs URL?

**Reply by alexey-milovidov** [reply]
https://clickhouse-builds.s3.yandex.net/0/4667ea78838148572713ea7b20d67585d1f59fb7/clickhouse_build_check/report.html

![Screenshot_20211025_102813](https://user-images.githubusercontent.com/18581488/138652813-894f72cc-f6f6-4962-ad7d-e01d6e7869a3.png)

**Reply by lingchengfenzi** [reply]
Thanks very much. Why my tips looks this? Does you install some plug-in or some settings?
<img width="1190" alt="截屏2021-10-26 下午5 04 43" src="https://user-images.githubusercontent.com/25588133/138993561-dce4ac06-518e-4538-82b4-19d797fe1c33.png">

**Reply by alexey-milovidov** [reply, marked as answer]
Hm, strange. I've checked - it is displaying correctly even without log in.

Maybe it's related to dark theme? Or you use some unusual client?
